studybuddies/src/app/components/login/login.component.ts

```diff
@@ -13,24 +17,50 @@ import {HttpClient} from '@angular/common/http'
 })
 export class LoginComponent implements OnInit {
 
-  user: User = new User();
-  loginError = null;
-  logged: boolean;
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private tokenStorage: TokenStorageService
+  ) 
+  {}
 
-  constructor(private loginService: LoginService, public router: Router, private cookieService: CookieService, public translate:TranslateService) { }
+  username = '';
+  password = '';
+  isLoggedIn = false;
+  isLoginFailed = false;
+  role = '';
 
-  ngOnInit() {
-    if (this.cookieService.get('auth_token') != "") {
-      this.logged = true;
-      this.loginService.getPrincipal()
-        .then (user => {
-          if(user.tutor == true)
-            this.router.navigate(['/tutor']);
-          else
-            this.router.navigate(['/student']);
-        });
-    } else
-      this.logged = false;
+  ngOnInit(): void {
+    if(this.tokenStorage.getUser()) {
+      this.isLoggedIn = true;
+      this.role = this.tokenStorage.getUser().role;
+    }
+    console.log(this.isLoggedIn)
+  }
+
+  login(){
+    if(!this.username) {
+      Swal.fire('Login', 'El nombre de usuario no es correcto', 'error')
+    }
+    if(!this.password) {
+      Swal.fire('Login', 'La contraseña indicada no es correcta', 'error')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
      Swal.fire("Login", 'La contraseña indicada no es correcta', 'error')
```